benchmarking/README.md

```diff
@@ -4,16 +4,93 @@ This directory contains sacred configuration files for benchmarking imitation's
 
 Configuration files can be loaded either from the CLI or from the Python API. The examples below assume that your current working directory is the root of the `imitation` repository. This is not necessarily the case and you should adjust your paths accordingly.
 
-## CLI
+To run a single benchmark from the command line:
 
 ```bash
 python -m imitation.scripts.<train_script> <algo> with benchmarking/<config_name>.json
 ```
+
 `train_script` can be either 1) `train_imitation` with `algo` as `bc` or `dagger` or 2) `train_adversarial`  with `algo` as `gail` or `airl`.
 
-## Python
+To run a single benchmark from Python add the config to your experiment:
 
 ```python
 ...
 ex.add_config('benchmarking/<config_name>.json')
 ```
+
+To generate the commands to run the entire benchmarking suite with multiple random seeds:
+
+```bash
+python experiments/commands.py \
+  --name=run0 \
+  --cfg_pattern=benchmarking/example_*.json \
+  --seeds 0 1 2 \
+  --output_dir=output
+```
+
+To run those commands in parallel:
+
+```bash
+python experiments/commands.py ... | parallel -j 8
+```
+
+To generate the commands for the Hofvarpnir cluster:
+
+```bash
+python experiments/commands.py \
+  --name=run0 \
+  --cfg_pattern=benchmarking/example_*.json \
+  --seeds 0 1 2 \
+  --output_dir=/data/output \
+  --remote
+```
+
+To run those commands pipe them into bash:
+
+```bash
+python experiments/commands.py ... | bash
```

Comment:
Some preference for commands actually being runnable (expand the ellipsis)

Reply:
Perhaps could just combine with above, or make it a two-step process (`| tee cmds_to_run`), verify, then `bash cmds_to_run`.

Reply:
I opted to just copy the command over for simplicity